src/main.rs

```diff
@@ -30,39 +30,48 @@ fn run_command(command: &str, work_dir: &Parent) -> Output {
         )
 }
 
-fn read_file(f: &FileArg) -> Result<String, std::io::Error> {
+fn die<A>(msg: String) -> A {
+    std::io::stderr().write_all((msg + "\n").as_bytes()).unwrap();
```

Comment:
how about prefixing all the messages like that? 
```suggestion
    std::io::stderr().write_all(("fatal: " + msg + "\n").as_bytes()).unwrap();
```

Reply:
Done.